model/src/test/kotlin/ProjectAnalyzerResultTest.kt

```diff
@@ -83,11 +85,13 @@ class ProjectAnalyzerResultTest : StringSpec({
             issues = listOf(issue7, issue8)
         )
 
-        val issues = result.collectIssues()
-        issues.size shouldBe 4
-        issues[Identifier("type:namespace:name:version")] shouldBe listOf(issue7, issue8)
-        issues[Identifier("type1:namespace1:name1:version1")] shouldBe listOf(issue3, issue4)
-        issues[Identifier("type2:namespace2:name2:version2")] shouldBe listOf(issue1, issue2)
-        issues[Identifier("type3:namespace3:name3:version3")] shouldBe listOf(issue5, issue6)
+        result.collectIssues() should containExactly(
```

Comment:
While at it we could remove the `over assertion` about the ordering.

Reply:
I think the map matcher does not assume any order, at least there is no `containExactlyInAnyOrder` variant.

Reply:
It's for maps, my bad! So no objections on this PR on my side.